rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/is_baseline_system_7.py

```diff
@@ -94,7 +94,7 @@ def is_baseline_system_7(rmi_b, hvac_b_id, terminal_unit_id_list, zone_id_list):
         and are_all_terminal_fans_null(rmi_b, terminal_unit_id_list)
         and are_all_terminal_types_VAV(rmi_b, terminal_unit_id_list)
     )
-
+    stop = 1
```

Comment:
What is the purpose of this line of code? `stop = 1`

Reply:
My apologies. This was just my stopping point for debugging. This was fixed.